src/pytest_run_parallel/plugin.py

```diff
@@ -133,6 +134,14 @@ def pytest_itemcollected(item):
         n_workers = 1
         item.add_marker(pytest.mark.parallel_threads(1))
 
+    if not hasattr(item, "obj"):
+        warnings.warn(
+            "Encountered PyTest item that is incompatible "
+            "with pytest-run-parallel, skipping pytest-run-parallel "
+            "monkeypatching"
```

Comment:
Maybe something like this would be better?

```suggestion
            "Encountered pytest item that is incompatible "
            "with pytest-run-parallel, skipping it"
```

Reply:
I went with something close to this, along with an f-string so the type of the item is in the error message:

```
../pytest-run-parallel/src/pytest_run_parallel/plugin.py:138
  /Users/goldbaum/Documents/pytest-run-parallel/src/pytest_run_parallel/plugin.py:138: UserWarning: Encountered pytest item with type '<class 'conftest.PyYAMLItem'> with no 'obj' attribute, which is incompatible with pytest-run-parallel. Tests using this item will not run in a thread pool
    warnings.warn(
```